execute_process_mass_data.py

```diff
@@ -1,6 +1,12 @@
 #!/usr/bin/env python
 
-from mof_array.pmf.process_mass_data import *
+from mof_array.pmf.process_mass_data import (read_output_data,
+                                            yaml_loader,
+                                            interpolate_data,
+                                            calculate_pmf,
+                                            create_bins,
+                                            bin_compositions,
+                                            plot_binned_pmf_array)
```

Comment:
This is totally legitimate (and maybe even recommended... see https://www.python.org/dev/peps/pep-0328/, learn something every day!) but I thought I'd check to make sure you know about line continuations. You could do the same thing like:

from mof_array.pmf.process_mass_data import read_output_data, \
                                            yaml_loader, \
                                            interpolate_data, \
                                            calculate_pmf, \
                                            create_bins, \
                                            bin_compositions

No need to change this, just making sure you know your other options!